core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -1736,9 +1738,11 @@ private[spark] object MapOutputTracker extends Logging {
 
   def validateStatus(status: ShuffleOutputStatus, shuffleId: Int, partition: Int) : Unit = {
     if (status == null) {
-      val errorMessage = s"Missing an output location for shuffle $shuffleId partition $partition"
+      // scalastyle:off line.size.limit
```

Comment:
Can we `split` it into multiple lines? To avoid using `// scalastyle:off line.size.limit`

Reply:
If the log is not long, one line makes it easier to read. I suggest we allow both styles.